src/FHIR/SMART/SMARTLaunchToken.php

```diff
@@ -0,0 +1,128 @@
+<?php
+
+namespace OpenEMR\FHIR\SMART;
+
+use OpenEMR\Common\Uuid\UuidRegistry;
```

Comment:
can't see where above class is being used here

Reply:
It's used by the SMARTLaunchController class.  It handles the serialization / deserialization of the 'launch' token that is used for the SMART launch-ehr capability.